client/contracts/bindings/consensus/consensus.go

```diff
@@ -69,15 +69,14 @@ func (tkn *mUSD) Mint(opts *accounts.TransactOpts, to common.Address, value *big
 
 func toBind(opts *accounts.TransactOpts) *bind.TransactOpts {
 	bindOpts := &bind.TransactOpts{
-		From:     opts.From,
-		Nonce:    opts.Nonce,
-		Value:    opts.Value,
-		GasPrice: opts.GasPrice,
-		Context:  opts.Context,
-		Signer:   bind.SignerFn(opts.Signer),
+		From:    opts.From,
+		Nonce:   opts.Nonce,
+		Value:   opts.Value,
+		Context: opts.Context,
```

Comment:
Will it possible to sync old kowala blockchain after this change?

Reply:
I don't think so - at least it's not trivial.

Reply:
Ok, so we need to try it on a dev network.